packages/autorest.python/samples/specification/azure-mgmt-test/test/azure-mgmt-test/azure/mgmt/test/_auto_rest_swagger_bat_array_service.py

```diff
@@ -31,12 +33,20 @@ class AutoRestSwaggerBATArrayService:  # pylint: disable=client-accepts-api-vers
     :vartype array: azure.mgmt.test.operations.ArrayOperations
     :param credential: Credential needed for the client to connect to Azure. Required.
     :type credential: ~azure.core.credentials.TokenCredential
-    :param base_url: Service URL. Default value is "http://localhost:3000".
+    :param base_url: Service URL. Required. Default value is "".
     :type base_url: str
     """
 
-    def __init__(self, credential: "TokenCredential", base_url: str = "http://localhost:3000", **kwargs: Any) -> None:
-        self._config = AutoRestSwaggerBATArrayServiceConfiguration(credential=credential, **kwargs)
+    def __init__(self, credential: "TokenCredential", base_url: str = "", **kwargs: Any) -> None:
```

Comment:
we should set the default to `_Unset: Any = object()` and use `base_url = _Unset` in the init line